data_steward/gcs_utils.py

```diff
@@ -31,8 +31,12 @@ def get_hpo_bucket(hpo_id):
     # TODO reconsider how to map bucket name
     bucket_env = 'BUCKET_NAME_' + hpo_id.upper()
     hpo_bucket_name = os.getenv(bucket_env)
+
     if hpo_bucket_name is None:
-        raise EnvironmentError()
+        # should not use hpo_id in message if sent to end user.  For now,
+        # only sent to alert messages slack channel.
+        raise OSError('No bucket name defined for hpo_id: {}'.format(hpo_id))
```

Comment:
@mark-velez Previously this was an empty message.  Is there a more generic error message we can report here?  I did say this should only be sent to the slack channel and never to the end user.  Using error handlers should ensure this.  Before checking in, I want your security perspective.

Reply:
I think principle of least privilege should prevail and your proposal to only send this information to the Slack channel sounds consistent with that. One could argue the service _does not have_ end-users (in the traditional sense), but I believe it makes sense to design conservatively, as though it did. Good thinking.